### ISSUE TITLE
Improvements for error message

### ISSUE THREAD

**Issue by selfthinker** [terrill/AU#7]
Under "12. Inaccessible form validation" on the list of accessibility issues, [the document says](https://github.com/terrill/AU/blob/master/issues.html#L165):

> Although the error message is clearly visible to sighted users, screen reader users are not informed that this message has appeared.

That is not completely true. Sighted users who zoom in won't see the message, mobile device users won't see the message, and sighted screen reader users will see the message.

I would change that to something like

> Although the error message is clearly visible to sighted users in desktop view who don't zoom into the page, screen readers don't announce that message, so users who cannot see will not be informed that this message has appeared.

You might also want to make the experience (deliberately) worse by not scrolling up when there is an error by [removing this line](https://github.com/terrill/AU/blob/33ce74f952f44d91eaa4e7abda08eee62b067e26/scripts/before-form.js#L15) (and then also changing the [description of that functionality](https://github.com/terrill/AU/blob/master/issues.html#L162)):

```
$(document).scrollTop(1);
```

When removing that functionality for sighted users in desktop view who don't zoom into the page, it "only" becomes a usability issue. But it becomes a worse accessibility issue than before (although not a WCAG fail) for screen magnifier users and users who use browser zoom.
